src/antidote_pb_txn.erl

```diff
@@ -71,7 +71,7 @@ process(#fpbatomicupdatetxnreq{ops = Ops}, State) ->
     end;
 
 process(#fpbsnapshotreadtxnreq{ops = Ops}, State) ->
-    lager:info("Testing txn interface.. Received snapshot read request  ~p", [Ops]),
```

Comment:
Because this outputs a message every time a transaction is received.. This will have severe performance penalty.